packages/http-server/src/server.ts

```diff
@@ -1,28 +1,73 @@
-const express = require('express');
+import { IFilesystemLoaderOpts } from '@stoplight/prism-core';
 import { createInstance } from '@stoplight/prism-http';
+import { IHttpMethod, TPrismHttpInstance } from '@stoplight/prism-http/types';
+import * as fastify from 'fastify';
+import { IncomingMessage, Server, ServerResponse } from 'http';
+
 import { getHttpConfigFromRequest } from './getHttpConfigFromRequest';
+import { IPrismHttpServer, IPrismHttpServerOpts } from './types';
+
+export const createServer = <LoaderInput = IFilesystemLoaderOpts>(
+  loaderInput: LoaderInput,
+  opts: IPrismHttpServerOpts<LoaderInput> = {}
+): IPrismHttpServer<LoaderInput> => {
+  const server = fastify<Server, IncomingMessage, ServerResponse>();
+
+  const prism = createInstance<LoaderInput>({
+    config: getHttpConfigFromRequest,
+    ...(opts.components || {}),
+  })(loaderInput);
+
+  server.all('*', {}, replyHandler(prism));
+
+  const prismServer: IPrismHttpServer<LoaderInput> = {
+    get prism() {
+      return prism;
+    },
+
+    get fastify() {
+      return server;
+    },
+
+    listen: async (...args) => {
+      return server.listen(...args);
+    },
+  };
+
+  return prismServer;
+};
+
+const replyHandler = (
+  prism: TPrismHttpInstance
+): fastify.RequestHandler<IncomingMessage, ServerResponse> => {
+  return async (request, reply) => {
+    const { req } = request;
+
+    try {
+      const response = await prism.process({
+        method: (req.method || 'get') as IHttpMethod,
+        url: {
+          path: req.url || '/',
+          query: request.query,
+        },
+        headers: request.headers,
+        body: request.body,
```

Comment:
Fastify parses body to object if it's application/json. It would most probably work, but I think we should ensure that we are sticking to raw string. The first example that comes to my head:
1. http-server receives request with content-length 100
2. http-server transforms body to object
3. forwarder calls axios with headers and body
4. axios transforms object to json leading to content-length mismatch

I believe there could be some more failures caused by parsing body.